Core/Base/ElasticDB.py

```diff
@@ -0,0 +1,96 @@
+########################################################################
+# $Id: $
+########################################################################
+
+""" ElasticDB is a base class used to connect an Elasticsearch database and manages 
+queries.
+"""
+
+__RCSID__ = "$Id$"
+
+from DIRAC                                       import gLogger, gConfig
+from DIRAC.Core.Utilities.ElasticSearchDB        import ElasticSearchDB
+from DIRAC.ConfigurationSystem.Client.Utilities  import getElasticDBParameters
+
+class ElasticDB( ElasticSearchDB ):
+
+  """
+  .. class:: ElasticDB
+
+  :param str dbHost: the host name of the Elasticsearch database
+  :param str dbPort: The port where the Elasticsearch database is listening
+  :param str clusterName: The name of the cluster.
+  """
+  __dbHost = None
+  __dbPort = None
+  __clusterName = ""
+  
+  ########################################################################
+  def __init__( self, dbname, fullName, debug = False ):
+    """ c'tor
+
+    :param self: self reference
+    :param str dbName: name of the database for example: MonitoringDB
+    :param str fullName: The full name of the database for example: 'Monitoring/MonitoringDB'
+    :param bool debug: save the debug information to a file 
+    """
+    
+    database_name = dbname
+    self.log = gLogger.getSubLogger( database_name )
+
+    result = getElasticDBParameters( fullName )
+    if not result['OK'] :
+      raise RuntimeError( 'Cannot get database parameters: %s' % result['Message'] )
+
+    dbParameters = result[ 'Value' ]
+    self.__dbHost = dbParameters[ 'Host' ]
+    self.__dbPort = dbParameters[ 'Port' ]
+    
+    ElasticSearchDB.__init__( self, self.__dbHost, self.__dbPort, debug = debug )
```

Comment:
well this also works in python3.
In addition python2.x super(className, self).__init__
python3.x super().__init__()